Feature/rocksdb validate sst (#1083)
arangodb/docs: 3.10/release-notes-new-features310.md

### RocksDB startup options

> There's a new startup option `--rocksdb.verify-sst` for validating the SST files currently

Reviewer: ```suggestion
The new `--rocksdb.verify-sst` startup option lets you validate the SST files currently
```

> There's a new startup option `--rocksdb.verify-sst` for validating the sst files currently
> contained in the database directory on startup. If set to true, on startup, all sst files
> in the `engine-rocksdb` directory of the database will be validated, then the program will
> finish execution. The default value is `false`.

Reviewer 2: ```suggestion
There's a new startup option `--rocksdb.verify-sst` for validating the SST files currently
contained in the database directory on startup. If set to `true`, on startup, all SST files
in the `engine-rocksdb` directory of the database will be validated, then the process will
finish execution. The default value is `false`.
```

> in the `engine-rocksdb` directory of the database will be validated, then the process will
> finish execution. The default value is `false`.

Reviewer: database directory path? of the database directory?
```suggestion
in the `engine-rocksdb` folder in the database directory are validated, then the process
finishes execution. The default value is `false`.
```
